optimum/habana/accelerate/state.py

```diff
@@ -173,6 +173,7 @@ def __init__(
                 if mixed_precision is None
                 else mixed_precision.lower()
             )
+            self.is_fp8_enabled = mixed_precision == "fp8"
```

Comment:
It is risky to set mixed_precision to "fp8", since it is being used at many other places. All we want to do is to set is_fp8_enabled = True here, Can we not do this in GaudiAccelerator __Init__ itself? Let's discuss this further.

Reply:
As discussed, we will proceed with this implementation.